src/inference/Inference.jl

```diff
@@ -485,6 +486,54 @@ end
 # Utilities  #
 ##############
 
+Base.@kwdef struct PriorExtractorContext{D,Ctx} <: AbstractContext
```

Comment:
I think there's nothing Turing.jl specific in this context and the helper function below? Then IMO DynamicPPL would be the right place to define them.

Reply:
@torfjelde what do you think? I have never played with upstream libs such as AbstractMCMC or DynamicPPL.

Reply:
Yeah, I agree with @devmotion here

Sorry, I should have been explicit about it; I just assumed it was a PR to DPPL :grimacing: